pkg/metrics/builtin/register.go

```diff
@@ -0,0 +1,15 @@
+package builtinmetrics
```

Comment:
The go coding style recommends to avoid tautological package names. `builtin` would be enough here. Also this way it would be easier to understand where `RegisterMetric()` comes from in the file `pkg/avx/collector.go` (currently it's not clear whether `builtinmetrics` is a package name or a global object defined in another file).

Reply:
Agree. I was struggling with the naming and this version was not very good. I've now re-worked the registration. PTAL.